### DISCUSSION TITLE
My gitbub dashboard is not getting updated.

### DISCUSSION THREAD

**Discussion by create-better** [community/community#119807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have been working on a project and commiting to my repo using git bash. The commits are visible in the repo but not in my dashboard. Why is this?

**Comment by MindOfBear** [comment]
Merge the main branch with the branch you are working on, this should fix your issue.

Only after merging the branch where you are working on with the main branch the commits will be visible in dashboard.

**Reply by create-better** [reply]
I tried this. But it did not work

**Comment by 0xabdulkhaliq** [comment]
Hi there @create-better, It sounds like the issue you're facing is that your commits are visible in the repository itself, but not on the main branch's file structure when viewed through the GitHub dashboard.

This can happen if you've been committing your changes to a different branch other than the main (or master) branch. Actually git repositories can have multiple branches, which allow developers to work on different features or versions of the codebase in parallel without affecting the main codebase.

When you commit changes to a branch other than the main branch, those changes will be reflected in that specific branch's commit history, but they won't be visible in the main branch's file structure until that branch is merged into the main branch.

To view the changes you've committed to the branch you've been working on, you can switch to that branch using the branch dropdown on the GitHub repository page. Alternatively, you can create a pull request from your branch to merge your changes into the main branch.

Once the pull request is reviewed and merged, your changes will become part of the main branch's file structure, and you'll be able to see them on the GitHub dashboard.

If you want to keep your changes on the main branch from the start, you should make sure you're committing and pushing to the main branch (often named "master" or "main") instead of creating a separate branch.

**Reply by create-better** [reply]
This does not work for @0xabdulkhalid

**Comment by manishpatel0** [comment]
Email mismatch: GitHub uses the committer email address to associate commits with your account. Make sure the email you configured in Git Bash (git config user.email) matches the email address on your GitHub account. You can check your local email with:
## git config user.email

**Reply by create-better** [reply]
I will try this. Thank you.
